data-generation/src/main/scala/utils/Benchmark.scala

```diff
@@ -30,20 +32,20 @@ case class Benchmark(writer: Writer, inSeconds: Boolean = false, textPrefix: Str
 
   def close(): Unit = writer.close()
 
-  def benchmarkSingle[T](function: => T, textPrefix: String = textPrefix): T = {
+  def benchmarkSingle[T](function: => T, textPrefix: String = textPrefix, customColumnValue: String = ""): T = {
```

Comment:
```suggestion
  def benchmarkSingle[T](function: => T, textPrefix: String = textPrefix, customColumnKeyValues: HashMap[String,String] ): T = {
```
Så sender man inn en HashMap når man benchmarker